source/includes/rest/_batch_api.md

```diff
@@ -274,8 +274,7 @@ response = client.track_events(events)
 // npm install drip-nodejs --save
 
 const client = require('drip-nodejs')({ token: YOUR_API_KEY, accountId: YOUR_ACCOUNT_ID });
-const payload = {
-  batches: [{
+const payload = [{
```

Comment:
The Javascript client wraps the payload in the `batches` key for us. https://github.com/DripEmail/drip-nodejs/blob/main/lib/events.js#L10